# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser-sets.inc
# Commit: a0daec0e7301
# Full Hash: a0daec0e73011a6b972d2bc87c1cf620d569af80
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-09-27 04:28:18
# Regressor Bug: 1582565
# File Overlap Count: 6
# Description:
#   Bug 1582565 - Log transactions into readable format. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46657
# ==============================================================================

diff -r 3bd01ec4a9e9 -r a0daec0e7301 browser/base/content/browser-sets.inc
--- a/browser/base/content/browser-sets.inc	Wed Sep 25 20:46:24 2019 +0000
+++ b/browser/base/content/browser-sets.inc	Thu Sep 26 16:43:05 2019 +0000
@@ -98,6 +98,7 @@
 #ifdef NIGHTLY_BUILD
     <command id="wrCaptureCmd" oncommand="gGfxUtils.webrenderCapture();"/>
     <command id="windowRecordingCmd" oncommand="gGfxUtils.toggleWindowRecording();"/>
+    <command id="wrTransactionLoggingCmd" oncommand="gGfxUtils.toggleTransactionLogging();"/>
 #endif
 #ifdef XP_MACOSX
     <command id="minimizeWindow"
@@ -331,6 +332,13 @@
     key="$" modifiers="control"
 #endif
     command="windowRecordingCmd"/>
+    <key id="key_transactionLoggingCmd"
+#ifdef XP_MACOSX
+    key="5" modifiers="control,shift"
+#else
+    key="%" modifiers="control"
+#endif
+    command="wrTransactionLoggingCmd"/>
 #endif
 #ifdef XP_MACOSX
     <key id="key_minimizeWindow"